index.bs

```diff
@@ -2251,6 +2251,69 @@ During the above process, the user agent SHOULD show some UI to the user to guid
 authorizing an authenticator. When <code>|options|.{{CredentialCreationOptions/mediation}}</code> is set to {{CredentialMediationRequirement/conditional}}, prominent modal UI should <i>not</i> be shown <i>unless</i> credential creation was previously consented to via means determined by the user agent.
 </div>
 
+#### Create Request Exceptions #### {#sctn-create-request-exceptions}
+
+[INFORMATIVE]
+
+[=[WRPS]=] can encounter a number of exceptions from a call to {{CredentialsContainer/create()|navigator.credentials.create()}}.
+Some exceptions can have multiple reasons for why they happened,
+requiring the [=[WRPS]=] to infer the actual reason based on their use of WebAuthn.
+
+Note: Exceptions that can be raised during processing of any [=WebAuthn Extensions=],
+including ones defined outside of this specification,
+are not listed here.
+
+The following {{DOMException}} exceptions can be raised:
+
+<dl>
+    :   {{AbortError}}
+    ::  The ceremony was cancelled by an {{AbortController}}.
+        See [[#sctn-abortoperation]] and [[#sctn-sample-aborting]].
+
+    :   {{ConstraintError}}
+    ::  Either {{residentKey}} was set to {{ResidentKeyRequirement/required}} and no available authenticator supported resident keys,
+        or {{AuthenticatorSelectionCriteria/userVerification}} was set to {{UserVerificationRequirement/required}} and no available authenticator could perform [=user verification=].
+
+    :   {{InvalidStateError}}
+    ::  The authenticator used in the ceremony recognized an entry in {{PublicKeyCredentialCreationOptions/excludeCredentials}}
+        after the user [=user consent|consented=] to registering a credential.
+
+    :   {{NotSupportedError}}
+    ::  No entry in {{PublicKeyCredentialCreationOptions/pubKeyCredParams}} had a {{PublicKeyCredentialDescriptor/type}} property of {{PublicKeyCredentialType/public-key}},
+        or the [=authenticator=] did not support any of the specified cryptographic parameters in {{PublicKeyCredentialCreationOptions/pubKeyCredParams}}.
+
+    :   {{SecurityError}}
+    ::  The [=effective domain=] was not a [=valid domain=],
+        or <code>{{PublicKeyCredentialCreationOptions/rp}}.{{PublicKeyCredentialRpEntity/id}}</code> was not equal to or a registrable domain suffix of the [=effective domain=].
+        In the case of [[#sctn-related-origins|related origin requests]],
+        the [=authenticator=] does not support it
+        or there were issues when [[#sctn-validating-relation-origin|validating related origins]].
+
+    :   {{TimeoutError}}
+    ::  The ceremony was cancelled by the user agent after exceeding the time limit permitted for the ceremony.
+        See [[#sctn-timeout-recommended-range]] for more information.
+
+    :   {{UnknownError}}
+    ::  The [=authenticator=] could not process the supplied options,
+        or encountered an error while creating the new credential.
```

Comment:
I'm not sure UnknownError is actually surfaced to the caller - the only occurrences in the spec are that the authenticator should "return an error code equivalent to UnknownError", but these errors are silently ignored by the 'If any authenticator returns an error status not equivalent to "[InvalidStateError](https://webidl.spec.whatwg.org/#invalidstateerror)"' case in `create()` and the "If any authenticator returns an error status" case in `get()`.

Reply:
Ah, nice insight to pull out of all of this, you're right. I'll remove `UnknownError` then, the RP will only ever get a `NotAllowedError` out of an authenticator raising an `UnknownError` (probably because the user will have to cancel out of the modal which of course becomes a `NotAllowedError`.)

Reply:
I've removed `UnknownError`

Reply:
Ping @nsatragno to make sure we interpreted this the same way browsers do?

Reply:
This seems correct: unknown errors should always be translated to `NotAllowedError`. Good catch!